README.md

```diff
@@ -83,6 +81,13 @@ Test the app with `Rspec` and `Capybara` frameworks.
 
 - Create database `chitter_manager_test` following the same steps as documented above.
 
+##Incomplete
+
+- The peeps displayed do not show the username - I would inject the User class into the Message class and use the username from the User class.
+- I would've liked to add more buttons on each page to swiftly move from one page to another
+- CSS styling
+- input a user_id foreign key into the messages table as each message has a user (one-to-many relationship)
+
 Notes on functionality:
 ------
 
```

Comment:
You could also add some instructions for future developers too:

For example, here's a snippet from the code rubric :)

```
$ git clone https://github.com/yourusername/chitter_challenge
$ bundle
$ rake setup
$ rake migrate
$ rspec
$ rackup
```